src/AppRouter.js

```diff
@@ -73,6 +75,21 @@ const AppRouter = () => (
                     <ApplicationsReviewPage />
                 </PageLayout>
             </ProtectedRoute>
+            <ProtectedRoute
+                exact
+                path="/company/offers/manage"
+                unauthorizedRedirectPath="/"
+                unauthorizedRedirectMessage="You are not allowed to access the my offers page"
+                authorize={(user) => !!(user?.company)}
+            >
+                <PageLayout
```

Comment:
PageLayout needs a unique `key` to work properly, as per the comment on the beginning of this file.

Use the route path for example

Reply:
I believe that is already implemented in line 86